src/domain/service.ts

```diff
@@ -243,20 +285,24 @@ export async function search(params: {
 
   if (params.states.length > 0) {
     query.state = {
-      $in: params.states as State[]
+      $in: params.states as State[],
     };
   }
 
   if (!!params.query) {
     query.$or = [];
-    query.$or.push({searchValues: new RegExp(params.query, 'gi')});
+    query.$or.push({ searchValues: new RegExp(params.query, 'gi') });
   }
 
+  // default sort by appId
   const sortObj: any = {};
-  params.sortBy.forEach(sb => {
-    sortObj[mapField(sb.field)] = sb.direction == 'asc' ?  1 : -1 ;
-  });
+  isEmpty(params.sortBy)
+    ? params.sortBy.forEach((sb) => {
+        sortObj[mapField(sb.field)] = sb.direction == 'asc' ? 1 : -1;
+      })
+    : (sortObj['appId'] = 1);
```

Comment:
to be sure I follow the intention here:

    line 299 checks if params.sortBy is an empty array
    if so, line 300 iterates over the empty array
    else, line 303 defines sortObj.appId as 1

I'm under the impression 303 should be the one with the ? (if the array is empty).
Maybe I'm misreading this?

Reply:
Nope you certainly are not...i meant to have a check for no sort values, but looks like i was thinking of the `sortObj` and used the empty check...i will change to check for an array length in order to bother mapping the sort array coming from params